account_reconcile_compassion/models/res_config.py

```diff
@@ -19,28 +18,28 @@ class AccountConfigSettings(models.TransientModel):
     currency exchange.
     """
 
-    _inherit = 'account.config.settings'
+    _inherit = "account.config.settings"
```

Comment:
'account.config.settings' should not exist.
This class should _inherit 'res.config.settings' and override the `get_values` and `set_values` methods.
Views need adaptations